### PR TITLE
Multiple changes during PTS 23

### PR DESCRIPTION
**Pull request by atoomic**
* start using perl 5.36
* update cpan snapshot
* use sereal for backend
* add `Cache-Control` header for CDN
* initial feature to exclude some files/path from a search

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -1,5 +1,10 @@
 #!/bin/sh
 
-#cp tools/pre-commit.pl .git/hooks/pre-commit
 [ -e .git/hooks/pre-commit ] ||  ln -s ../../tools/pre-commit.pl .git/hooks/pre-commit
-cpanm --installdeps --with-develop --with-recommends .
+cpanm App::cpm
+cpanm Carton::Snapshot
```
Comment (oalders) [inline install.sh:5]: Could probably already use `cpm` at this point, since you're not using the snapshot file here.

Comment (atoomic) [reply install.sh:5]: note: this script is not used by the Dockerfile at this pont but for setting up a dev en on a freshly installed perl version

File: lib/GrepCpan/std.pm [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+sub import {
+
+    # auto import strict and warnings to our caller
+
+    warnings->import();
+    strict->import();
+
+    require feature;
+    feature->import( ':5.30', 'signatures' );
```
Comment (oalders) [inline lib/GrepCpan/std.pm:14]: I guess this could be 5.36, but maybe not if you want it to be runnable outside of Docker.

**Review by oalders — APPROVED** [review]

File: lib/GrepCpan/std.pm (hunk 2 above)
Comment (atoomic) [reply lib/GrepCpan/std.pm:14]: yeap both could work

**Comment by oalders** [comment]
@atoomic I will leave the merge up to you.

**Comment by atoomic** [comment]
Yeap waiting for the k8 cluster being deployed
Then would have to build the image

**Pull request merged**